docs/descendant-manager.md

```diff
@@ -3,8 +3,17 @@
 ### Purpose
 
 The Descendant Manager indexes content using pre-defined field traversal paths in node type .yml
-files to determine relationships. These relationships are grouped into three
-types:
+files to determine relationships. 
+
+Specific use cases:
+- Determine the service that the page is in so that we can show the right “log in” links in header that will be defined on that service page.
```

Comment:
```suggestion
- Determine the service that the page is in so that we can show the correct “login” links in the header, which we define on that service page.
```